src/components/bridge/BridgeSelection.vue

```diff
@@ -5,6 +5,41 @@
         <span>{{ $t('bridge.selectBridge') }}</span>
       </div>
       <div class="container--selection">
+        <div class="column--selection">
+          <button :disabled="!isEnableMinatoBridge">
+            <component
+              :is="isEnableMinatoBridge ? 'router-link' : 'div'"
+              :to="buildCcipBridgePageLink()"
+              class="button--bridge"
+            >
+              <div class="row--logo-bg">
+                <div class="img--logo-bg">
+                  <img
+                    class="img--logo-soneium"
+                    :src="require('src/assets/img/chain/soneium-color.svg')"
+                    alt="soneium"
+                  />
+                </div>
+              </div>
+              <div class="row--bridge-title">
+                <div class="text--bridge-tag">
+                  <q-chip outline>
+                    {{ $t('bridge.ccipMinatoBridge.tag') }}
+                  </q-chip>
+                </div>
+                <span class="text--bridge-title">{{ $t('bridge.ccipMinatoBridge.title') }}</span>
+                <div class="box--text-bridge">
+                  <span class="text--bridge">
+                    {{ $t('bridge.ccipMinatoBridge.text') }}
+                  </span>
+                </div>
+              </div>
+            </component>
+          </button>
+          <p v-if="!isShibuyaEvm" class="text--bridge-details">
+            {{ $t('bridge.ccipMinatoBridge.text2') }}
```

Comment:
Use a meaningful variable name instead of text and text2

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1416/commits/3f3438f5e3ac392293ecce0d71be877fee95b0a2) commit.